src/tabcenter.js

```diff
@@ -8,10 +8,7 @@ function TabCenter() {
 
 TabCenter.prototype = {
   async init() {
-    const darkTheme = (await browser.storage.local.get({
-      darkTheme: false
-    })).darkTheme;
-    this.toggleTheme(darkTheme);
+    this._detectDarkTheme();
```

Comment:
I think it is not interfering for other code and obviously (not too far and not complicated). If the `_detectDarkTheme` was modified later, modifiers will see the `async`.